### PR TITLE
Disconnect repo button (closes #49)

### LINKED ISSUE #49: Add disconnect button
Add a button to allow disconnecting your site from the current git repo. This button must be visible in conf. step 2 and status.

### PR DESCRIPTION
**Pull request by bogdanpetrea**

### CODE REVIEW

**Comment by krodyrobi** [comment]
Looks good :+1:, even kept the verbose naming convention :D

File: gitium/inc/class-gitium-menu.php [hunk 1]
```diff
@@ -66,4 +77,15 @@ public function show_message() {
...
 		}
 	}
+
+	protected function show_disconnect_repository_button() {
+		?>
+		<form name="gitium_form_disconnect" id="gitium_form_disconnect" action="" method="POST">
+			<?php
+				wp_nonce_field( 'gitium-admin' );
+		  ?>
+			<input type="submit" name="GitiumSubmitDisconnectRepository" value='<?php _e( 'Disconnect from repo', 'gitium' ); ?>' class="button secondary" onclick="return confirm('<?php _e( 'Are you sure you want to disconnect from the remote repository?', 'gitium' ); ?>')"/>&nbsp;
```
Comment (rciorba) [inline gitium/inc/class-gitium-menu.php]: I don't really know what the coding style says about dropping out of php mode and back in again.
I personally find it a bit hard to read. @Marius786 do you have any thoughts about this?

Comment (Marius786) [inline gitium/inc/class-gitium-menu.php]: @rciorba I don’t think WordPress is particularly consistent either way. At the moment, it depends on what is most readable for that situation.

Comment (rciorba) [inline gitium/inc/class-gitium-menu.php]: ok, fair enough.. LGTM then

**Comment by calind** [comment]
@bogdanpetrea Please squash all the commits and add a new entry to the change log.

**Comment by Marius786** [comment]
LGTM

**Pull request merged**